OpenImis.ePayment/Escape/Payment/Data/GepgUtility.cs

```diff
@@ -43,40 +43,45 @@ public GepgUtility(IHostingEnvironment hostingEnvironment, IConfiguration Config
             
         }
 
-        public String CreateBill(IConfiguration Configuration, string OfficerCode, string PhoneNumber, string BillId, decimal ExpectedAmount, List<InsureeProduct> products)
+        public String CreateBill(IConfiguration Configuration, string OfficerCode, string PhoneNumber, string BillId, decimal ExpectedAmount, List<PaymentDetail> policies)
         {
 
             DataHelper dh = new DataHelper(Configuration);
 
             List<BillItem> items = new List<BillItem>();
 
-            BillItem item = new BillItem()
+            if ( policies.Count==1 )
             {
-                BillItemRef = "ImisPolicy",
-                BillItemAmt = Convert.ToDouble(ExpectedAmount),
-                BillItemEqvAmt = Convert.ToDouble(ExpectedAmount),
-                BillItemMiscAmt = 0,
-                UseItemRefOnPay = "N",
-                GfsCode = Configuration["PaymentGateWay:GePG:GfsCode:0"]
-            };
-
-            items.Add(item);
-
-            //foreach (var product in products)
-            //{
+                BillItem item = new BillItem()
```

Comment:
This is not needed. Just use the foreach loop even if there is only one policy in the list. However, we need to check if there is at least one policy, if not return error.

Reply:
ok. clear for me!